### DISCUSSION TITLE
Feature of assets (cpu, ram, ssd)

### DISCUSSION THREAD

**Discussion by follet-git** [snipe/snipe-it#11950] [Q&A]
Hello. I created an asset - a laptop. I need to add information about its feature, like CPU, ram and SSD capacity.
- I can't use 2 fieldsets
- I do not need to create components because in the components tab I use only components that I buy separately
Any idea?

**Comment by ej-exerp** [comment]
You can create a customer field under "Settings" and then create a fieldset and register it on the asset model. That's what I have done.
